### ISSUE TITLE
[BUG] Enum's __class__ etc. are pulled in when inherited_members is true

### ISSUE THREAD

**Issue by yi2020** [mkdocstrings/pytkdocs#107]
**Describe the bug**
I've set inherited_memberse to true in my selector to pull in members that are inherited from classes that I've built. Essentially, I have a class Mergeable, that has some methods and attributes, and many classes inherit from it. However, this is also applying to Enum and is pulling in noise.

<img width="215" alt="Screen Shot 2021-05-17 at 11 36 28 AM" src="https://user-images.githubusercontent.com/10076447/118539717-9c011180-b704-11eb-93af-130b0e4d189e.png">
<img width="749" alt="Screen Shot 2021-05-17 at 11 41 10 AM" src="https://user-images.githubusercontent.com/10076447/118539843-c8b52900-b704-11eb-9383-1de91912e6a8.png">

**To Reproduce**
Use mkdocs+mkdocstring on a Python file that has an Enum in it, and set inherited_members to true.

**Expected behavior**
The __class__ and similar aspects shouldn't be collected. There's no point in documenting these are part of mkdocs.

**System (please complete the following information):**
- `pytkdocs` version 0.11.1
- Python version: 3.8.2
- OS: MacOS

**Comment by yi2020** [comment]
BTW - workaround was to filter anything that starts with `__`, but the default behavior shouldn't require this.

**Comment by pawamoy** [comment]
Whether we should filter out members starting with `__` is up to discussion :slightly_smiling_face: 
Personally I want my documented special methods to appear in the docs (`__init__`). But I agree it can quickly become noisy with inherited members.

**Comment by yi2020** [comment]
Agree, though maybe special handling there? Does it make sense to pull `__bool__` and `__class__`, for example? `__init__` may be a special one, and only if it's non empty?

**Comment by pawamoy** [comment]
The project is now archived.
